docs/identify.rst

```diff
@@ -0,0 +1,40 @@
+==============================
+Identifying Spectrum1D Formats
+==============================
+
+``specutils`` provides a convenience function,
+`~specutils.io.registers.identify_spectrum_format`, which attempts to guess the
+`~specutils.Spectrum1D` file format from the list of registered formats, and
+essentially acts as a wrapper on `~astropy.io.registry.identify_format`.
+
+This function is useful for identifying a spectrum file format without the need to
+read it in with Spectrum1d's `~specutils.Spectrum1D.read` method.  It returns the best guess as to a
+valid format from the list of ``Formats`` as given by `~astropy.io.registry.get_formats`.
```

Comment:
Perhaps this could spell out still more clearly that it is using the same identification methods as `Spectrum1D.read` – I doubt many users will be familiar with the internals of `~astropy.io.registry`.

Reply:
I've updated the docs here with another sentence or two.  Let me know what you think.